graylog2-server/src/main/java/org/graylog2/indexer/elasticsearch/GlobalTimeoutClient.java

```diff
@@ -84,7 +84,15 @@ public GlobalTimeoutAction(final Action<Request, Response, RequestBuilder> actio
         @Override
         public TransportRequestOptions transportOptions(Settings settings) {
             final TransportRequestOptions result = super.transportOptions(settings);
-            return result.timeout() == null ? result.withTimeout(timeout) : result;
+            if (result.timeout() == null) {
```

Comment:
`TransportRequestOptions#withTimeout()` has been removed in Elasticsearch 2.2.0 and we now need to use the corresponding builder which looks quite ugly using a conditional expression.